views/popups.pug

```diff
@@ -95,6 +95,15 @@
                 label
                   input.enableCommunityAds(type="checkbox")
                   | Show community events
+        .card  
+          .card-body
+            .card-title
+              h4 Keybinding
+            .form-group(role="group")
+              .checkbox
+                label
+                  input.useVim(type="checkbox")
+                  | VIM (ctrl+c to switch to normal mode)
```

Comment:
Is it not possible to use `Esc` to toggle back and forth? I think my poor brain would explode if I couldn't use esc (or caps lock as I map it...)